packages/react-aria/src/hooks/useARIAButton.stories.tsx

```diff
@@ -35,6 +35,7 @@ export const Anchor = (args: DefaultArgs) => {
   const props = useARIAButton(
     {
       as: 'a',
+      href: '/',
```

Comment:
should we be defaulting `href` values ? it's quite valid to declare an anchor without that attribute

Reply:
oh I see it's already done in the render of the slot, so you're not changing much

Reply:
Yeah, I agree that it seems a little weird.

Reply:
Yeah I'm just moving it up from where it used to be below. The type checking based on the `as` prop only works when the type-specific types are specified in the same object that specifies the `as` prop.

Below, the compiler doesn't know that `slots.root` is actually `a` (because it could also be `div`, `span`, or `button`), so the `href` prop doesn't work there.